packages/world/src/modules/keysintable/KeysInTableHook.sol

```diff
@@ -63,11 +63,41 @@ contract KeysInTableHook is StoreHook {
     // NOOP
   }
 
-  function onBeforeSetField(bytes32 tableId, bytes32[] memory keyTuple, uint8, bytes memory, FieldLayout) public {
+  function onBeforeSpliceStaticData(bytes32, bytes32[] calldata, uint48, uint40, bytes calldata) public pure {
     // NOOP
   }
```

Comment:
if we add these method stubs to `StoreHook` with a `NotImplemented` error, we could remove these here